i hate when you can't do `blah -h | less` because the program 
'blah' dumps its help text to stderr ... it forces you to do 
crap like `blah -h 2>&1 | less` ... what a pita.

this patch tells netcat to holler() at stdout instead of stderr 
when displaying the usage information.

--- netcat.c
+++ netcat.c
@@ -205,2 +205,3 @@
 USHORT o_verbose = 0;
+USHORT o_holler_stderr = 1;
 unsigned int o_wait = 0;
@@ -231,4 +232,5 @@
 {
+  FILE *o_holler_out = (o_holler_stderr ? stderr : stdout);
   if (o_verbose) {
-    fprintf (stderr, str, p1, p2, p3, p4, p5, p6);
+    fprintf (o_holler_out, str, p1, p2, p3, p4, p5, p6);
 #ifdef HAVE_BIND
@@ -236,5 +238,5 @@
       if (h_errno > 4)		/* oh no you don't, either */
-	fprintf (stderr, "preposterous h_errno: %d", h_errno);
+	fprintf (o_holler_out, "preposterous h_errno: %d", h_errno);
       else
-	fprintf (stderr, h_errs[h_errno]);	/* handle it here */
+	fprintf (o_holler_out, h_errs[h_errno]);	/* handle it here */
       h_errno = 0;				/* and reset for next call */
@@ -245,4 +247,4 @@
     } else			/* xxx: do something better?  */
-      fprintf (stderr, "\n");
-    fflush (stderr);
+      fprintf (o_holler_out, "\n");
+    fflush (o_holler_out);
   }
@@ -2338,2 +2340,3 @@
   o_verbose = 1;
+  o_holler_stderr = 0;
   holler ("[v1.10]\n\
